pkg/daemon/update.go

```diff
@@ -595,6 +595,7 @@ func calculatePostConfigChangeActionFromMCDiffs(diffFileSet []string) (actions [
 	}
 	dirsPostConfigChangeActionReloadCrio := []string{
 		constants.SigstoreRegistriesConfigDir,
+		constants.CrioPoliciesDir,
```

Comment:
(In https://github.com/cri-o/cri-o/pull/8212, the policies are re-read on every operation, and nothing is cached, so a reload is not currently necessary. But it doesn’t hurt.)

Reply:
@yuqi-zhang could you take a look? I am unsure where the mco API is for adding this directory to no action list?

Reply:
We set the default policies now via https://github.com/openshift/machine-config-operator/blob/7c5ae75515fe373e3eecee711c18b07d01b5e759/pkg/apihelpers/apihelpers.go#L23

Reply:
moved this logic to [pkg/apihelpers/apihelpers.go](https://github.com/openshift/machine-config-operator/blob/7c5ae75515fe373e3eecee711c18b07d01b5e759/pkg/apihelpers/apihelpers.go#L23)